### ISSUE TITLE
Unable to use Demucs-GUI versions beyond 1.1a2

### ISSUE THREAD

**Issue by EtemT** [CarlGao4/Demucs-Gui#81]
Hello! I have a strange issue where GUI versions released after 1.1a2 do not seem to open on my machine. 1.1a2 works just fine, but for all versions after, when I try running the Demucs-GUI.exe file, I'll get the command prompt window appear for a second, then disappear, but the program itself not open. I've tried the following troubleshooting steps:

- Running DemucsGUI as administrator
- Clearing the AppData\Roaming\demucs-gui folder
- Downloading the latest versions of Python and PyTorch (from what I can see this shouldn't make a difference but thought I'd give it a go.)
- Tested both the CPU and CUDA versions, both not working (1.1a2 CUDA is what I'm using at the moment).
- Moving the installation directory to the same place I have 1.1a2_cuda.
- Doing all of the above after restarting my computer

Below is the log file, which lists that a File couldn't be found, but I'm not sure what file I'm missing or what it's failing to load exactly. Am I missing something obvious or silly? Apologies if this is not the right place for it!

> 2024-07-13 16:42:04,922 (GuiMain.py) (Line 2049) [INFO] : Python version: 3.11.9 | packaged by conda-forge | (main, Apr 19 2024, 18:27:10) [MSC v.1938 64 bit (AMD64)]
2024-07-13 16:42:04,922 (GuiMain.py) (Line 2050) [INFO] : Demucs GUI version: 1.2
2024-07-13 16:42:04,967 (GuiMain.py) (Line 2051) [INFO] : System: Windows-10-10.0.19045-SP0
2024-07-13 16:42:04,967 (GuiMain.py) (Line 2052) [INFO] : Architecture: 64bit
2024-07-13 16:42:04,967 (GuiMain.py) (Line 2053) [INFO] : CPU: Intel64 Family 6 Model 158 Stepping 10, GenuineIntel
2024-07-13 16:42:04,967 (GuiMain.py) (Line 2054) [INFO] : CPU count: 12
2024-07-13 16:42:04,972 (GuiMain.py) (Line 2055) [INFO] : System memory: 34290499584 (31.936GB)
2024-07-13 16:42:04,976 (GuiMain.py) (Line 2058) [INFO] : System free memory: 24795066368 (23.092GB)
2024-07-13 16:42:05,503 (GuiMain.py) (Line 2062) [INFO] : System swap memory: 9232711680 (8.599GB)
Traceback (most recent call last):
  File "GuiMain.py", line 2069, in <module>
  File "<frozen importlib._bootstrap>", line 1176, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1147, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 690, in _load_unlocked
  File "PyInstaller\loader\pyimod02_importers.py", line 419, in exec_module
  File "find_device_win.py", line 199, in <module>
  File "shared.py", line 340, in Popen
  File "shared.py", line 108, in wrapped_Popen
  File "subprocess.py", line 1026, in __init__
  File "subprocess.py", line 1538, in _execute_child
FileNotFoundError: [WinError 2] The system cannot find the file specified

**Comment by CarlGao4** [comment]
Please follow the instructions on https://github.com/CarlGao4/Demucs-Gui/tree/debug-win64

**Comment by CarlGao4** [comment]
Please press <kbd>Win</kbd> + <kbd>X</kbd>, and select `PowerShell (Adminstrator)`, and paste the following command into the terminal window:

```PowerShell
[Environment]::SetEnvironmentVariable("Path", [Environment]::GetEnvironmentVariable("Path", "Machine") + ";C:\Windows\System32\Wbem", "Machine")
```

Then press <kbd>Return</kbd>, and restart your computer

**Comment by EtemT** [comment]
Thank you very much for the response - that fixed the issue!

**Comment by CarlGao4** [comment]
Theoretically, this has been fixed in 1.3a1
